_docs/2.0.0/usage/menas-api.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
+title: Usage - Menas API
+version: '2.0.0'
+categories:
+    - '2.0.0'
+    - usage
+---
+
+- [Export Entity](#export-entity)
+  - [Example Import Response for Schema](#example-import-response-for-schema)
+- [Entity Import](#entity-import)
+  - [JSON payload](#json-payload)
+  - [Example Export Response for Schema](#example-export-response-for-schema)
+
+
+### Export Entity
+
+This feature exports a single entity from Menas. All exports are versionless, except for connected entities. They are kept as is, but need to be validated before import, that they match.
```

Comment:
They are kept as is however would need to be validated before import to ensure that they match.

Reply:
to ensure they match ( remove that)